orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -337,11 +339,13 @@ class TaskController {
 
     if (!expand) {
       allPipelines.each { pipeline ->
+        if (pipeline.trigger instanceof PipelineTrigger) {
+          ((PipelineTrigger) pipeline.trigger).parentExecution.getStages().clear()
```

Comment:
So, interesting story, this line definitely gets called but also definitely seems to not make any difference in the response, as I will still see all the stages in the `trigger.parentExecution`... still trying to figure out why that is the case...